cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +634,47 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
```

Comment:
I don't see the std out monitored in this test?

Reply:
Stdout is logging into `evm.log` and using `LogValidator` finding expected pattern i.e. `verbosity` value and asserting it in UI as well.